File: src/1. Layers/1.3 Domain/Well.Domain/CleanDelivery.cs
namespace PH.Well.Domain
{
    public class CleanDelivery:Entity<int>
    {
        public CleanDelivery()
        {
        }

        public string  RouteNumber { get; set; }
        public string DropId { get; set; }
        public string InvoiceNumber { get; set; }
        public string AccountCode { get; set; }
        public string AccountName { get; set; }
        public string JobStatus { get; set; }

    }
}

File: src/1. Layers/1.1 Presentation/Well.Dashboard/app/clean/cleanDelivery.ts
import { BaseDelivery } from '../shared/baseDelivery';

export class CleanDelivery extends BaseDelivery {
    public routeNumber: string;
    public routeDate: Date;
    public dropId: string;
    public invoiceNumber: number;
    public accountName: string;
    public jobStatus: string;
    public action: string;
    public deliveryDate: Date;
    public accountId: string;
    public canAction: boolean;
} 